web/app/routes.php

```diff
@@ -20,7 +20,9 @@
 SimpleRouter::get('/PasswordChange', [\Praktikant\Praktikum\Controller\ChangePasswordController::class, 'index'], ['as' => 'ChangePW.get']);
 SimpleRouter::get('/PasswordChangeController', [\Praktikant\Praktikum\Controller\PasswordChangeController::class, 'index'], ['as' => 'ChangePW2.get']);
 SimpleRouter::get('/PasswordChange/PasswordChangeController', [\Praktikant\Praktikum\Controller\PasswordChangeController::class, 'index'], ['as' => 'ChangePW2.get']);
+SimpleRouter::get('/RealVerifyController', [\Praktikant\Praktikum\Controller\RealVerifyController::class, 'index'], ['as' => 'RealVerifyController.get']);
```

Comment:
controller nicht in die route schreiben. die routen sollten benutzerfreundlich sein